main/src/ca/uwaterloo/flix/language/phase/HtmlDocumentor.scala

```diff
@@ -71,9 +71,9 @@ object HtmlDocumentor {
     */
   val LibraryGitHub: String = "https://github.com/flix/flix/blob/master/main/src/library/"
 
-  def run(root: TypedAst.Root)(implicit flix: Flix): Unit = {
+  def run(root: TypedAst.Root, includedModules: Option[List[String]])(implicit flix: Flix): Unit = {
```

Comment:
Instead of the vague type, `Option[List[String]`, I think what we want is `Set[Symbol.ModuleSym]`.

This will ensure that we are actually taking about modules of the right format.

I think it should also help simplify some of the later code.